test/gui/tst_sharing/test.feature

```diff
@@ -111,6 +111,16 @@ Feature: Sharing
             | TestUser1 |
             | TestUser2 |
 
+    Scenario: autocomplete offers a list of users followed by a list of groups
+        Given user "admin" has been created on the server with default attributes and without skeleton files
+        And user "Alice" has uploaded file with content "ownCloud test text file" to "textfile.txt" on the server
+        And user "Alice" has set up a client with default settings
+        When the user opens the sharing dialog of "textfile.txt" using the client-UI
+        And the user searches for collaborator with autocomplete characters "Adm" using the client-UI
+        Then the following users should be listed as suggested collaborators:
+            | user  |
+            | Admin |
+            | Admin |
```

Comment:
We were told that sharing dialog will be dropped. So do we still want to cover the test for it?
See https://github.com/owncloud/client/pull/10959#issuecomment-1602479049
CC @TheOneRing

Reply:
That's just an oc10 scenario